scenarios/http_downstream/tests/logic_http_downstream_001.py

```diff
@@ -0,0 +1,167 @@
+import http.client, time, json, os, ssl, logging, requests
+
+from utils.fluent_bit_manager import FluentBitManager
+from utils.network import find_available_port
+from server.otlp_server import  data_storage
+
+logger = logging.getLogger(__name__)
+
+def create_connection_tlsON(server, port, cafile):
```

Comment:
let's not use camel case, e.g: `def create_connection_tlsON()` -> `def create_connection_tls_on()`

Reply:
all requested changes have been applied.